packages/rest-typings/src/v1/users/UsersListStatusParamsGET.ts

```diff
@@ -0,0 +1,44 @@
+import Ajv from 'ajv';
+
+import type { PaginatedRequest } from '../../helpers/PaginatedRequest';
+
+const ajv = new Ajv({
+	coerceTypes: true,
+});
+
+export type UsersListStatusParamsGET = {
+	status: 'active' | 'all' | 'deactivated' | 'pending';
+	roles: string[];
+	searchTerm: string;
```

Comment:
Your properties here are required, but on the AJV schema they're not. Which one is the right one?

Reply:
They are not required, let me fix

Reply:
Thanks 🚀

Reply:
Sorry, `status` is required

Reply:
Oh, if its required, then add is as `required: ['status']` on the ajv schema, before additionalProperties

assuming that's the only required one, remember to update the roles & searchTerm props on this type to put them as optional